docs/flashing.md

```diff
@@ -119,6 +119,36 @@ Flashing sequence:
 3. Flash a .hex file
 4. Reset the device into application mode (may be done automatically)
 
+## bootloadHID
+
+BootloadHID is a USB bootloader for AVR microcontrollers. The uploader tool requires no kernel level driver on Windows and can therefore be run without installing any DLLs.
+
+To ensure compatibility with the bootloadHID bootloader, make sure this block is present your `rules.mk`:
+
+    # Bootloader
+    #     This definition is optional, and if your keyboard supports multiple bootloaders of
+    #     different sizes, comment this out, and the correct address will be loaded
+    #     automatically (+60). See bootloader.mk for all options.
```

Comment:
And does this really add 60 bytes to the compiled size?

Reply:
Not sure, its a copy-paste from <https://github.com/qmk/qmk_firmware/blob/master/quantum/template/ps2avrgb/rules.mk>. I know removing sizes has been mentioned previously.

Reply:
I should also note, its the same as Caterina, DFU, HalfKay